libs/pipeline.py

```diff
@@ -10,6 +10,13 @@
 import us
 
 
+def fips_to_location_id(fips: str) -> str:
+    """Converts a FIPS code to a locationID"""
+    if len(fips) == 2:
+        iso1: us  # iso2:us-ca#fips:06087
```

Comment:
This feels incomplete?

Reply:
oops, that's from another branch. fixed with 2a8fcdd